### ISSUE TITLE
Race setup submision after qualifying closes

### ISSUE THREAD

**Issue by JonBeagles** [christopherh0rn/f1-revisited-bugs#67]
Per "UAT #12 - Edit Strategy after submitting #41" the drivers need to be able to submit a race setup after qualifying closes.

We discussed this being 48 hours afterwards to give me time to get the race setup if I needed it.  Everyone is saying they can't get in to amend this though?

There are at least 2 drivers who have been trying to submit race setups that have been getting this token error message I put in the recent bug report FYI - they now can't even get onto the page to be able to try and submit a setup (totally not their fault, as far as I can make out).

**Comment by christopherh0rn** [comment]
I had thought we had tested this functionality in UAT and it worked satisfactorily?

**Comment by JonBeagles** [comment]
Thanks mate :)

Session finish time for 11pm tonight if possible - I can then give the 48 hour notice at that point.

If you don't see this tonight, then by all means 11pm on whatever day you do see this!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by christopherh0rn** [comment]
Session closing time has been set to 11pm tonight so should be accessible from 23:00 to submit/amend race setups.

**Comment by JonBeagles** [comment]
Yep looks spot on - just tested on Rach's profile and it works :)
